### DISCUSSION TITLE
Mark, come over to Orca!

### DISCUSSION THREAD

**Discussion by PhilBaz** [smartavionics/Cura#248] [General]
Hey Mark.

Hope you are doing great.

Not sure if you have had the chance to try Orca slicer yet. I find it has become my main slicer, next to your branch of Cura, which is still invaluable when I need the maximum level of control. 

But I wanted to say you should give Orca a try and maybe even have a look at their git. They are releasing steadily, and bringing in interesting developments like the scarf seam. Maybe it would be something you'd be interested in.

Thanks for continuing to support the best version of Cura!

Phil

**Comment by smartavionics** [comment]
Hi Phil,

Thanks for your kind comments. I have tried Orca and it does look interesting. TBH, as you've probably gathered from my recent lack of activity, I'm not actively developing any new slicer stuff, just fiddling here and there and fixing the odd bug. I'm not even doing much in the way of 3d printing right now as most of my printing was used for my business (Smart Avionics Ltd.) which is now pretty much dormant.

All the best,

Mark
